### PR TITLE
adds support for optionally creating secret

### PR DESCRIPTION
**Pull request by bit-herder**
So i did this so the chart supports creating a dockerconfig secret by filling out values, but if you don't, it wont break whats there now. 

If you hate it lmk, we can just create the secret manually as well. It was more of an exercise of learn2helm than anything else.

### CODE REVIEW

**Review by vaxinate — APPROVED** [review]
NOICE

**Comment by bit-herder** [comment]
I bumped the chart version and ran the package script. I think that covers it, but im gonna wait for @jwietelmann to review before I merge since I'm new to this. 

Unsure if there are any other steps to make this live

**Comment by jwietelmann** [comment]
replacing with #3

**Pull request closed without merging**
